### ISSUE TITLE
false MD026 alarm when header ends with an emoji

### ISSUE THREAD

**Issue by RochaStratovan** [markdownlint/markdownlint#527]
## Description

The `mdl` tool is raising a false _MD026 Trailing punctuation in header_ when a header ends with an emoji.

For example, the following valid markdown is incorrectly flagged by `mdl`

```md
## :warning: WARNING :warning:
```

### Environment

Ubuntu 20.04.6 LTS

**MDL Version**

```bash
$ mdl --version
0.13.0
```

### Expected Behavior

No alarm should be raised in this situation.

### Actual Behavior

An alarm is raised.

## Replication Case

Create a file with the example shown above and run `mdl` against it.
